islandora_xacml_editor.drush.inc

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/**
+ * @file
+ * Drush hooks.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function islandora_xacml_editor_drush_command() {
+  $items = array();
+
+  $items['islandora_xacml_editor_apply_policy'] = array(
+    'aliases' => array('ixeap'),
+    'description' => dt('Apply XACML policy to target object.'),
+    'bootstrap' => DRUSH_BOOTSTRAP_DRUPAL_LOGIN,
+    'examples' => array(
+      'Apply policy.xml to \'islandora:57\', use traversal to target child objects.' => 'drush -v --user=1 islandora_xacml_editor_apply_policy --policy=/tmp/policy.xml --pid=islandora:57 --traversal',
+    ),
+    'options' => array(
+      'policy' => array(
+        'description' => dt('The path to an XML file containing the XACML policy configuration to be applied. It is expected that this policy file be generated from the UI\'s XACML Editor.'),
+        'required' => TRUE,
+      ),
+      'pid' => array(
+        'description' => dt('The PID of the object to apply the policy configuration to.'),
+        'required' => TRUE,
+      ),
+      'traversal' => array(
+        'description' => dt('Optional. Whether to apply policies to the target object\'s children (shallow traversal is not supported for collection objects). Disabled by default.'),
```

Comment:
I know this is hard to explain in simple words, but anything that helps explaining drush command users what the difference between enabled/not enabled would help here. Not a hard requirement of course, just feel it could be needed since not everyone reads code/knows what happens currently via UI. Thanks

Reply:
Good point, I'll reword and push it shortly.